src/context.ts

```diff
@@ -68,24 +77,74 @@ export function commands<C extends Context>() {
         };
 
         ctx.getNearestCommand = (commands, options) => {
-            if (ctx.msg?.text) {
-                commands = ensureArray(commands);
-                const results = commands.map((commands) => {
-                    const userInput = ctx.msg!.text!.substring(1);
-                    const result = fuzzyMatch(userInput, commands, {
-                        ...options,
-                        language: !options?.ignoreLocalization
-                            ? ctx.from?.language_code
-                            : undefined,
-                    });
-                    return result;
-                }).sort((a, b) => (b?.similarity ?? 0) - (a?.similarity ?? 0));
+            if (!ctx.has(":text")) {
+                throw new Error(
+                    "cannot call `ctx.getNearestCommand` on an update with no `text`",
+                );
+            } else {
```

Comment:
We don't need an else here. Throw already stops the execution.

Reply:
changed in [da8568a](https://github.com/grammyjs/commands/pull/31/commits/da8568ab522a01f156e6c8ecb40b60a8090ec7fc)